versions/3.1.0.md

```diff
@@ -2689,10 +2689,10 @@ This object MAY be extended with [Specification Extensions](#specificationExtens
 
 The discriminator object is legal only when using one of the composite keywords `oneOf`, `anyOf`, `allOf`.
 
-In OAS 3.0, a response payload MAY be described to be exactly one of any number of types:
+A payload MAY be described to be exactly one of any number of types:
```

Comment:
I suggest removing the word "response", because the discriminator can be used in the request and response.